web/template/layout/data-loss-warning-dialog.gohtml

```diff
@@ -1,40 +1,36 @@
 {{ define "data-loss-warning-dialog" }}
-    <div id="dialog-overlay" class="app-dialog-overlay govuk-!-display-none" tabindex="-1"></div>
-
-    <div id="dialog"
-        class="app-dialog govuk-!-display-none"
-        role="dialog"
+    <dialog id="data-loss-dialog"
+        class="app-dialog"
         aria-labelledby="dialog-title"
         aria-describedby="dialog-description"
         aria-modal="true">
 
-        <h2 id="dialog-title" class="govuk-heading-l" tabindex="0">{{ tr . "youHaveUnsavedChanges" }}</h2>
-        <p id="dialog-description" class="govuk-body" tabindex="0">{{ trHtml . "unsavedChangesDialogContent" }}</p>
+        <h2 id="dialog-title" class="govuk-heading-l">{{ tr . "youHaveUnsavedChanges" }}</h2>
+        <p id="dialog-description" class="govuk-body">{{ trHtml . "unsavedChangesDialogContent" }}</p>
 
         <div class="govuk-button-group">
-            <button type="button" class="govuk-button govuk-button--secondary" data-module="govuk-button" aria-label="{{ tr . "backToPage" }}">{{ tr . "backToPage" }}</button>
+            <button autofocus type="button" class="govuk-button govuk-button--secondary" data-module="govuk-button" aria-label="{{ tr . "backToPage" }}">{{ tr . "backToPage" }}</button>
             <a href="{{ link . (global.Paths.TaskList.Format .LpaID) }}" class="govuk-button govuk-button--warning">{{ tr . "continueWithoutSaving" }}</a>
         </div>
-    </div>
+    </dialog>
 
-    <div id="language-dialog"
-        class="app-dialog govuk-!-display-none"
-        role="dialog"
+    <dialog id="language-dialog"
+        class="app-dialog"
         aria-labelledby="language-dialog-title"
         aria-describedby="language-dialog-description"
         aria-modal="true">
```

Comment:
MDN says,

> <dialog> elements invoked by the showModal() method implicitly have [aria-modal="true"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-modal)

I was hoping you'd be able to remove all of these, but it doesn't look like that is the case